app/views/users/show.html.erb

```diff
@@ -1,9 +1,11 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
+  <%= invite_friend(@user) %>
```

Comment:
Please change the color of the "Invite to friendship" because it's black and easy to miss (match the green style you have on the other view)